src/syrupy/__init__.py

```diff
@@ -41,11 +41,6 @@ def __import_extension(value: Optional[str]) -> Any:
         raise argparse.ArgumentTypeError(e)
 
 
-def __default_extension_option(value: Optional[str]) -> Any:
-    __import_extension(value)
```

Comment:
This PR undoes this line, which was called out in [a comment from #667](https://github.com/tophat/syrupy/pull/667/files#r1059134028). Do you remember why it was necessary? Just for better DX maybe, failing fast?

Reply:
Yes to fail faster.

Reply:
How do you feel about losing that fail faster functionality, making it fail later, after pytest has done all its `pythonpath` bootstrapping?

Reply:
The trade-off makes sense to me. I'm good with this 👍